jupyterhub/services/service.py

```diff
@@ -160,6 +160,11 @@ class Service(LoggingConfigurable):
     def managed(self):
         """Am I managed by the Hub?"""
         return bool(self.command)
+    
+    @property
+    def kind(self):
+        """Am I managed (as text)"""
```

Comment:
Perhaps `"""Am I self managed?"""`. I'm not sure that the suggestion is better, but `as text` wasn't clear to me.

Or maybe: `"""Determine kind of service. True, self managed. False, externally managed."""`